decisions/0003-github-labels-decluttering.md

```diff
@@ -0,0 +1,85 @@
+# Use of Labels in OSCAL Project(s)
+
+Date: 02/17/2023
+
+## Status
+
+Feedback Requested
```

Comment:
Non-blocking: I will not make this mistake again like I did on my own PR for the previou ADRs, once we are ready to merge this we switch it as needed.

```suggestion
Approved
```